ClosedXML_Sandbox/app.config

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0"?>
-<configuration>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0"/>
-  </startup>
-</configuration>
+<?xml version="1.0"?>
```

Comment:
This one too...? What changed in this file?

Reply:
I do not know, the git showed me that the files were changed and did not allow to roll back the changes.